### PR TITLE
Fix mistral callback failure when result contains unicode

### PR DESCRIPTION
**Pull request by m4dcoder**
When result of action execution contains unicode, Mistral callback fails and leads to orphaned workflow execution.

### CODE REVIEW

File: contrib/runners/mistral_v2/tests/unit/test_mistral_v2_callback.py [hunk 1]
```diff
@@ -142,6 +173,94 @@ def test_callback_handler_with_result_as_list_str(self):
...
+            '12345',
+            state='SUCCESS',
+            output='{"a": "\\\\u4ec0\\\\u9ebc"}'
+        )
+
+    @mock.patch.object(
+        action_executions.ActionExecutionManager, 'update',
+        mock.MagicMock(return_value=None))
+    def test_callback_handler_with_result_as_dict_with_unicode_type(self):
+        self.callback_class.callback('http://127.0.0.1:8989/v2/action_executions/12345', {},
```
Comment (Kami) [inline contrib/runners/mistral_v2/tests/unit/test_mistral_v2_callback.py]: It would be good to test unicode for all the various type (string, list, dict) and also add a test case with actual unicode string (`💩`) in addition to existing tests with the unicode escape sequences.

Comment (Kami) [reply contrib/runners/mistral_v2/tests/unit/test_mistral_v2_callback.py]: Besides that, LGTM :+1:

**Comment by Kami** [comment]
Added v2.4.1 milestone since I assume this will / should go into v2.4.1.

**Comment by armab** [comment]
👍 

There are also some failed Unit tests.

File: contrib/runners/mistral_v2/tests/unit/test_mistral_v2_callback.py (hunk 1 above)
Comment (m4dcoder) [reply contrib/runners/mistral_v2/tests/unit/test_mistral_v2_callback.py]: @Kami added more unit tests as requested
